modules/combined/doc/content/newsletter/2018_10_04.md

```diff
@@ -156,6 +156,26 @@ influence matrix conditioning and solution accuracy, and generally
 should be avoided if possible.
 
 
+## Add FEProblem-derived objects via Actions
+
+Custom Problems are advanced MOOSE objects which are useful in several
+different scenarios, including coupling to external codes
+(`ExternalProblem`) and solving eigenvalue problems
+(`EigenProblem`). Prior to the addition of this `Action`, using a
+custom Problem required users to declare a `Problem` block in their
+input file with a custom `type =` field, and this process has now been
+streamlined.  A test of this new capability has been added in
```

Comment:
Add *Code developers can now use an Action to add the derived FEProblem object programatically. This can often make the user interface nicer because the desired FEProblem type can typically depend on information in the input file. Forcing users to provide the correct FEProblem type can be removed.*